sympy/functions/special/bessel.py

```diff
@@ -216,11 +217,44 @@ def _eval_rewrite_as_bessely(self, nu, z, **kwargs):
     def _eval_rewrite_as_jn(self, nu, z, **kwargs):
         return sqrt(2*z/pi)*jn(nu - S.Half, self.argument)
 
+    def _eval_as_leading_term(self, x, cdir=0):
+        nu, z = self.args
+        arg = z.as_leading_term(x)
+        if x in arg.free_symbols:
+            return z**nu
+        else:
+            return self.func(*self.args)
+
     def _eval_is_extended_real(self):
         nu, z = self.args
         if nu.is_integer and z.is_extended_real:
             return True
 
+    def _eval_nseries(self, x, n, logx, cdir=0):
+        from sympy.series.order import Order
+        nu, z = self.args
+
+        # In case of powers less than 1, number of terms need to be computed
+        # separately to avoid repeated callings of _eval_nseries with wrong n
+        try:
+            _, exp = z.leadterm(x)
+        except (ValueError, NotImplementedError):
+            return self
+
+        if exp.is_positive:
+            newn = ceiling(n/exp)
+            r = z._eval_nseries(x, n, logx, cdir).removeO()
```

Comment:
```suggestion
            r = (z/2)._eval_nseries(x, n, logx, cdir).removeO()
```
This way 2 and 4 can be deleted from the denominators.